CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1703](https://github.com/nix-rust/nix/pull/1703))
 - Added `ptrace::read_user` and `ptrace::write_user` for Linux.
   (#[1697](https://github.com/nix-rust/nix/pull/1697))
+- Added `from_sigset_t_unchecked()` to `signal::SigSet`.
```

Comment:
Can you add a link to this PR, similar to other items (squashed into the current commit)?